source/documentation/services/sonarcloud.html.md.erb

```diff
@@ -0,0 +1,54 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: MoJ SonarCloud Account
+last_reviewed_on: 2023-07-28
+review_in: 3 months
+---
+# SonarCloud
+
+SonarCloud is a powerful cloud-based code analysis and continuous code quality service provided by SonarSource. Designed to enhance your development process, SonarCloud automatically analyzes your code for bugs, security vulnerabilities, code smells, and maintainability issues.
+
+Some of the key features include:
+
+* Code Quality Analysis: Identify coding issues and bugs through static code analysis.
+* Security Vulnerability Detection: Scan your code for potential security issues.
+* Code Smells Detection: Uncover design issues to improve code maintainability and performance.
+* Technical Debt Management: Prioritize and address critical technical debt issues.
+* CI/CD Integration: Integrate SonarCloud into your CI/CD pipeline.
+* Pull Request Analysis: Catch issues in code changes before merging into the main codebase.
+
+## Logging into SonarCloud
+
+To access Sonarcloud, use your existing GitHub account for seamless login through the "Login With GitHub" option on the main page.
```

Comment:
Does this require digital justice email on the account or just an account with the Org?

Reply:
It requires a GitHub account